setup.py

```diff
@@ -25,22 +25,22 @@
 # therein for more information.
 sys.path.insert(0, os.path.dirname(__file__))
 
-try:
-    import versioneer
-    version_config = versioneer.get_version()
-    cmdclass = versioneer.get_cmdclass()
-except ImportError:
-    print("WARNING: failed to import versioneer, falling back to no version for now")
-    version_config = "0.1.0"  # Fallback version
-    cmdclass = {}
+# try:
+#     import versioneer
+#     version_config = versioneer.get_version()
+#     cmdclass = versioneer.get_cmdclass()
+# except ImportError:
+#     print("WARNING: failed to import versioneer, falling back to no version for now")
+#     version_config = "0.4.0"  # Fallback version
+#     cmdclass = {}
 
 # Ensure the version is PEP 440 compliant
-if '+' in version_config:
-    version_config = version_config.split('+')[0]
+# if '+' in version_config:
+#     version_config = version_config.split('+')[0]
 
 if __name__ == "__main__":
     setup(
         name="lincbrain",
-        version="0.1.0",
-        cmdclass=cmdclass,
+        version="0.9.0",
```

Comment:
This is the value that is bumped to create a new semantic version in PyPI for now during the build and distribute phases